css-typed-om/Overview.bs

```diff
@@ -2903,6 +2903,44 @@ depending on the property they came from:
         return "currentcolor".
     2. Otherwise, return the result of serializing the <<color>> value.
 
+: 'display'
+::
+    1. Let |values| initially be the empty [=list=].
+    2. If 'display-outside' or 'display-inside' is specified:
+        1. Append value of 'display-outside' to |values|.
+        2. Append value of 'display-inside' to |values|.
+        3. Goto step 7.
+    3. If 'display-listitem' is specified:
+        1. Append ''list-item'' to |values|.
+        2. If 'display-outside' is specified, serialize it and append it to |values|.
+        3. If ''flow'' or ''flow-root'' is specified, append it to |values|.
+        4. Goto step 7.
+    4. If 'display-internal' is specified:
+        1. Append its value to |values|.
+        2. Goto step 7.
+    5. If 'display-box' is specified:
+        1. Append its value to |values|.
+        2. Goto step 7.
+    6. If 'display-legacy' is specified:
+        1. Append its value to |values|.
+        2. Goto step 7.
+    7. If |values| is:
+        1. ''block flow'', return 'block'.
```

Comment:
|values| is a list of strings, not a sequence of CSS values. Needs to be `«"block", "flow"»`, per [the Infra Standard](https://infra.spec.whatwg.org/#lists).